File: java/src/test/java/com/murex/RomanNumeralsTest.java
package com.murex;

import org.junit.Ignore;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class RomanNumeralsTest {

    @Ignore
    @Test
    public void acceptance_test() {
        int arabic = 4746;
        String roman = "MMMMDCCXLVI";
        assertEquals(roman, RomanNumerals.arabicToRoman(arabic));
    }
}

File: python/tests/roman_numerals_test.py
import pytest

from roman_numerals.roman_numerals import arabic_to_roman


class TestRomanNumerals:

    @pytest.mark.skip(reason="test currently disabled")  # Comment or remove this line to enable this test case
    def test_acceptance_test(self) -> None:
        arabic = 4746
        roman = 'MMMMDCCXLVI'
        assert (roman == arabic_to_roman(arabic))
